### DISCUSSION TITLE
Unable to change Instance Type in GitHub Codespaces

### DISCUSSION THREAD

**Discussion by Rishit-dagli** [community/community#26916] [Codespaces]
<p>Originally reported at <a href="https://github.com/MicrosoftDocs/vscodespaces/issues/790" rel="nofollow noopener">MicrosoftDocs/vscodespaces#790</a>, ported to Support Community as suggested by <a class="mention" href="/lostintangent">@lostintangent</a></p>
<p>Issue Type:  <strong>Feature Request</strong></p>
<p>While using <a href="https://github.com/codespaces" rel="nofollow noopener">GitHub Codespaces</a> and creating a new codespace I have no option where I could change the Instance Type. As of now, I believe an instance type is pre-specified for you maybe while reviewing a PR a “Basic” instance is created, and so on. It would be helpful to have this option customizable by the user while creating a codespace.</p>
<p>Though I have not yet tried it out I believe that this could maybe be customized with the Preferences Sync, but it would be great to have this right while creating one.</p>

**Comment by matthewisabel** [comment, marked as answer]
<p>Thanks for the suggestion! It’s on the roadmap for the future once we’re closer to GA and have billing place.</p>
